exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricDescriptorStrategy.java

```diff
@@ -0,0 +1,61 @@
+package com.google.cloud.opentelemetry.metric;
+
+import com.google.api.MetricDescriptor;
+
+import java.util.Collection;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.function.Consumer;
+
+/**
+ * The strategy for how to handle metric descriptors.
+ */
+public interface MetricDescriptorStrategy {
+    /**
+     * Determines what to do with metrci descriptors.
+     * 
+     * @param batchDescriptors The set of metrics being exported in a batch.
+     * @param client           A consumer that will ensure metric descriptors are
+     *                         registered to cloud monitoring.
+     */
+    void exportDescriptors(Collection<MetricDescriptor> batchDescriptors, Consumer<MetricDescriptor> export);
```

Comment:
Could we use `Iterable<MetricDescriptor>` instead of `Collection<...>` here?

Reply:
Scala habits.... yes we likely can